### PR TITLE
feat(cb2-15350b): added const array for abandoned tests

### PR DESCRIPTION
**Pull request by cb-cs**
## VTx - VTG/VTP12 can be generated from abandoned test's

This adds a new const array to the test type directory, listing all test type ids that would warrant a VTG or VTP12 being generated, should they be abandoned.

Related issue: [CB2-15350](https://dvsa.atlassian.net/browse/CB2-15350)

## Before submitting (or marking as "ready for review")

- [X] Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification?
- [X] Have you performed a self-review of the code
- [X] Have you have added tests that prove the fix or feature is effective and working
- [ ] Did you make sure to update any documentation relating to this change?

### CODE REVIEW

File: packages/cvs-common/src/classes/testTypes/Constants.ts [hunk 1]
```diff
@@ -209,3 +209,99 @@ export const CENTRAL_DOCS_TEST = {
 		'195',
 	],
 };
+
+export const ABANDONED_TEST = {
```
Comment (naathanbrown) [inline packages/cvs-common/src/classes/testTypes/Constants.ts:213]: Is this the right name? Maybe it is, but can other tests be abandoned but not make a VTG12 etc?

Comment (cb-cs) [reply packages/cvs-common/src/classes/testTypes/Constants.ts:213]: It's just naming convention as I wanted to amalgamate both the VTP12 and VTG12 test type ids.

This list will be used to determine whether or not the test type id on the test result is valid to produce the certificate, assuming the flag is enabled.

Comment (naathanbrown) [reply packages/cvs-common/src/classes/testTypes/Constants.ts:213]: Yeah I think so, if I looked at this in a year I'd like to know that it's for a certain document, not just those are tests that can be abandoned, which is how it reads.

Comment (naathanbrown) [reply packages/cvs-common/src/classes/testTypes/Constants.ts:213]: I've gone for `VTG_VPT_12` in a few places, doesn't have to match but just a suggestion.

Comment (cb-cs) [reply packages/cvs-common/src/classes/testTypes/Constants.ts]: No problem, updated.

**Review by naathanbrown — APPROVED** [review]

**Review by naathanbrown — APPROVED** [review]

**Review by m-coslett — APPROVED** [review]

**Pull request merged**